src/components/portfolio/DetailsMainPositions.tsx

```diff
@@ -0,0 +1,180 @@
+  GridList,
+  GridListTile,
+  Button,
+} from '@material-ui/core';
+import ArrowRightIcon from '@material-ui/icons/ArrowRight';
+import { useTranslation } from 'react-i18next';
+import { Position } from './DetailsTypes';
+import ValueOverName from './ValueOverName';
+
+// stylesheet for the positions section
+const useStyles = makeStyles(({ palette }: Theme) =>
+  createStyles({
+    titleContainer: {
+      display: 'flex',
+      marginBottom: '2rem',
+    },
+    titleWrapper: {
+      marginRight: '1rem',
+    },
+    sectionTitle: {
+      margin: 0,
+      color: palette.primary.contrastText,
+      // TODO use theme fontsize and weight
+      fontSize: '2.25rem',
+      fontWeight: 400,
+      whiteSpace: 'nowrap',
+    },
+    lineWrapper: {
+      display: 'flex',
+      width: '100%',
+      // TODO: use theme color
+      borderColor: '#EEF1FB',
+    },
+    line: {
+      width: '100%',
+      alignSelf: 'center',
+      paddingLeft: '2%',
+    },
+    button: {
+      textTransform: 'none',
+      // TODO: use theme color
+      color: '#EEF1FB',
+      padding: 0,
+    },
+    gridListWrapper: {
+      width: '100%',
+      margin: '0 -16px',
+    },
+    gridList: {
+      width: '100%',
+      height: 'auto',
+      maxHeight: '50rem',
+    },
+    card: {
+      backgroundColor: palette.primary.main,
+      padding: '0 8px',
+    },
+    cardContentUpper: {
+      marginBottom: '2rem',
+    },
+    cardTitle: {
+      color: palette.primary.contrastText,
+      fontSize: '1.2rem',
+      fontWeight: 600,
+    },
+    cardSubtitle: {
+      // TODO: use theme color
+      color: '#EEF1FB',
+      fontSize: '1rem',
+      fontWeight: 500,
+    },
+    cardContentLower: {
+      display: 'flex',
+      justifyContent: 'space-around',
+    },
+    cardAction: {
+      padding: '16px',
+    },
+  })
+);
+
+// type declarations
+type DetailsMainPositionsProps = {
+  positions: Position[];
+};
+
+// returns the details page header
+const DetailsMainPositions: React.FC<DetailsMainPositionsProps> = ({
+  positions,
+}) => {
+  const [posState] = React.useState(positions);
+  const classes = useStyles();
+  const { t } = useTranslation();
+
+  // TODO: no hard coded colors
+  // takes a percent value and converts it to a color
+  function convertPercentToColor(val: number): string {
+    return val < 0 ? '#D64745' : '#50E2A8';
+  }
+
+  return (
+    <div>
+      <div className={classes.titleContainer}>
+        <div className={classes.titleWrapper}>
+          <h2 className={classes.sectionTitle}>
+            {t('portfolio.details.positionsTitle')}
+          </h2>
+        </div>
+        <div className={classes.lineWrapper}>
+          <hr className={classes.line} />
+        </div>
+      </div>
+      <div className={classes.gridListWrapper}>
+        <GridList
+          cellHeight="auto"
+          cols={4}
+          spacing={32}
+          className={classes.gridList}
+          style={{ margin: '0 auto' }}
+        >
```

Comment:
An additional scrollbar appears if the height of the `GridList` exceeds 800px. I think limiting the height of this part is a good idea, but I'd prefer a "show more/less" button instead of nested scrollbars.

Reply:
To be done in #106